www/netj00/Semestralni Prace/odkazy.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="cs">
+<head>
+    <meta charset="UTF-8">
+    <title>Úvodní Strana</title>
+    <link rel="stylesheet" type="text/css" href="data/css/layout.css">
+    <link rel="icon" href="data/images/UO_icon.png" type="image/gif" sizes="16x16">
+</head>
+<body>
+<section>
+    <div class="nav">
```

Comment:
`<nav>`?

Reply:
nav

Reply:
nezapomen kod preformatovat, ve tvem IDE to jde taky pmoci nejake kombinace klaves, nebo pravy klik a vybrat z menu